### PR TITLE
Updating docs, adding and fixing examples for Microsoft Teams

### PR DESCRIPTION
**Pull request by RamjotSingh**

### CODE REVIEW

File: api-reference/beta/api/chatmessage-post.md [hunk 1]
```diff
@@ -748,6 +748,315 @@ Content-type: application/json
 }
 ```
 
+### Example 7 : @mention a channel in a channel message
+
+#### Request
+The following is an example of the request. Refer to [list channels](../api/channel-list.md) for getting list of channels in a team.
```
Comment (JarbasHorst) [inline api-reference/beta/api/chatmessage-post.md:754]: For consistency and readability, we introduce the subject before we provide the link. In this case, I recommend: 

- For information about how to get a list of channels in a team, see \[List channels\]\(../api/channel-list.md\).

File: api-reference/beta/api/chatmessage-post.md [hunk 2]
```diff
@@ -748,6 +748,315 @@ Content-type: application/json
...
+        }
+    ],
+    "reactions": []
+}
+```
+
+### Example 9 : @mention a tag in a channel message
+
+#### Request
+The following is an example of the request. Refer to [list tag](../api/teamworktag-list.md) for getting list of tags in a team.
```
Comment (JarbasHorst) [inline api-reference/beta/api/chatmessage-post.md:963]: For consistency and readability, we introduce the subject before we provide the link. In this case, I recommend: 

- For information about how to get a list of tags in a team, see \[List teamworkTags\]\(../api/teamworktag-list.md\).

File: api-reference/v1.0/api/chatmessage-post.md [hunk 3]
```diff
@@ -746,6 +746,215 @@ Content-type: application/json
 }
 ```
 
+### Example 7 : @mention a channel in a channel message
+
+#### Request
+The following is an example of the request. Refer to [list channels](../api/channel-list.md) for getting list of channels in a team.
```
Comment (JarbasHorst) [inline api-reference/v1.0/api/chatmessage-post.md:752]: For consistency and readability, we introduce the subject before we provide the link. In this case, I recommend: 

- For information about how to get a list of channels in a team, see \[List channels\]\(../api/channel-list.md\).

File: api-reference/v1.0/api/chatmessage-post.md [hunk 4]
```diff
@@ -746,6 +746,215 @@ Content-type: application/json
...
+
+> Note: `conversationIdentityType` needs to be set to `channel` to @mention a channel.
```
Comment (JarbasHorst) [inline api-reference/v1.0/api/chatmessage-post.md:754]: Property/Resource names should have bold formatting applied. For readability, consider also replacing "needs to" with "must". Suggested revision:

- \*\*conversationIdentityType\*\* must be set to \`channel\` to @mention a channel.

File: api-reference/beta/api/chatmessage-post.md [hunk 5]
```diff
@@ -748,6 +748,315 @@ Content-type: application/json
 }
 ```
 
+### Example 7 : @mention a channel in a channel message
+
+#### Request
+The following is an example of the request. Refer to [list channels](../api/channel-list.md) for getting list of channels in a team.
+
+> Note: `conversationIdentityType` needs to be set to `channel` to @mention a channel.
```
Comment (JarbasHorst) [inline api-reference/beta/api/chatmessage-post.md:756]: Property/Resource names should have bold formatting applied. For readability, consider also replacing "needs to" with "must". Suggested revision:

- \*\*conversationIdentityType\*\* must be set to \`channel\` to @mention a channel.

File: api-reference/beta/api/chatmessage-post.md [hunk 6]
```diff
@@ -748,6 +748,315 @@ Content-type: application/json
...
+    "reactions": []
+}
+```
+
+### Example 8 : @mention a team in a channel message
+
+#### Request
+The following is an example of the request.
+
+> Note: `conversationIdentityType` needs to be set to `team` to @mention a team.
```
Comment (JarbasHorst) [inline api-reference/beta/api/chatmessage-post.md:860]: Property/Resource names should have bold formatting applied. For readability, consider also replacing "needs to" with "must". Suggested revision:

- \*\*conversationIdentityType\*\* must be set to \`team\` to @mention a team.

File: api-reference/v1.0/api/chatmessage-post.md [hunk 7]
```diff
@@ -746,6 +746,215 @@ Content-type: application/json
...
+    "reactions": []
+}
+```
+
+### Example 8 : @mention a team in a channel message
+
+#### Request
+The following is an example of the request.
+
+> Note: `conversationIdentityType` needs to be set to `team` to @mention a team.
```
Comment (JarbasHorst) [inline api-reference/v1.0/api/chatmessage-post.md:858]: Property/Resource names should have bold formatting applied. For readability, consider also replacing "needs to" with "must". Suggested revision:

- \*\*conversationIdentityType\*\* must be set to \`team\` to @mention a team.

**Review by JarbasHorst — CHANGES_REQUESTED** [review]

File: api-reference/beta/api/chatmessage-post.md [hunk 8]
```diff
@@ -748,6 +748,315 @@ Content-type: application/json
...
+                    "conversationIdentityType": "channel"
+                }
+            }
+        }
+    ]
+}
+```
+
+#### Response
+
```
Comment (JarbasHorst) [inline api-reference/beta/api/chatmessage-post.md:788]: For consistency, please, add "The following is an example of the response." after \#\#\#\# Response

File: api-reference/beta/api/chatmessage-post.md [hunk 9]
```diff
@@ -748,6 +748,315 @@ Content-type: application/json
...
+                }
+            }
+        }
+    ],
+    "reactions": []
+}
+```
+
+#### Response
+
```
Comment (JarbasHorst) [inline api-reference/beta/api/chatmessage-post.md:893]: For consistency, please, add "The following is an example of the response." after \#\#\#\# Response

File: api-reference/beta/api/chatmessage-post.md [hunk 10]
```diff
@@ -748,6 +748,315 @@ Content-type: application/json
...
+                    "displayName": "TestTag"
+                }
+            }
+        }
+    ]
+}
+```
+
+#### Response
+
```
Comment (JarbasHorst) [inline api-reference/beta/api/chatmessage-post.md:994]: For consistency, please, add "The following is an example of the response." after \#\#\#\# Response

File: api-reference/v1.0/api/chatmessage-post.md [hunk 11]
```diff
@@ -746,6 +746,215 @@ Content-type: application/json
...
+                    "conversationIdentityType": "channel"
+                }
+            }
+        }
+    ]
+}
+```
+
+#### Response
+
```
Comment (JarbasHorst) [inline api-reference/v1.0/api/chatmessage-post.md:786]: For consistency, please, add "The following is an example of the response." after \#\#\#\# Response

File: api-reference/v1.0/api/chatmessage-post.md [hunk 12]
```diff
@@ -746,6 +746,215 @@ Content-type: application/json
...
+                }
+            }
+        }
+    ],
+    "reactions": []
+}
+```
+
+#### Response
+
```
Comment (JarbasHorst) [inline api-reference/v1.0/api/chatmessage-post.md:891]: For consistency, please, add "The following is an example of the response." after \#\#\#\# Response

**Review by JarbasHorst — CHANGES_REQUESTED** [review]

**Comment by JarbasHorst** [comment]
@Lauragra this PR is ready for final review.

**Review by Lauragra — APPROVED** [review]

**Pull request merged**